test/podcast/create.test.js

```diff
@@ -0,0 +1,33 @@
+const test = require('tape')
+const axios = require('axios')
+const testAPI = require('../testing-api')
+
+test('POST /podcast', async t => {
+  // t.equal(2 + 2, 4, 'addition is working!')
+
+  const api = testAPI()
+
+  // create a new podcast!
+  const result = await axios.post('http://localhost:3001/podcast', {
+    title: 'my buttcast adventure!',
+    description: 'season 1, episode 1'
+  })
+  // console.log('result', result.data)
+
+  t.equal(typeof result.data.id, 'string', 'returns a podcast id!')
```

Comment:
Finally we test to check that what comes back from the server looks legit.
(console.log out result.data) to see what comes back.

`t.equal(A, B, comment)` 
- A = actual value from running a function, 
- B = what we expected it to be, 
- comment = a human friendly comment